straxen/plugins/acqmon_processing.py

```diff
@@ -6,69 +6,68 @@
-                                 help = 'Minimum hit threshold in ADC*counts above baseline'),
-                    strax.Option('baseline_samples_aqmon', default = 10, track = False,
-                                 help = 'Number of samples to use at the start of the pulse to determine the baseline'))
-
+@strax.takes_config(strax.Option('hit_min_amplitude_aqmon', default=50, track=False,
+                                 help='Minimum hit threshold in ADC*counts above baseline'),
+                    strax.Option('baseline_samples_aqmon', default=10, track=False,
+                                 help='Number of samples to use at the start of the pulse to determine the baseline'))
 class AqmonHits(strax.Plugin):
     """ Find hits in acquisition monitor data. These hits could be 
         then used by other plugins for deadtime calculations, 
         GPS SYNC analysis, etc.
     """
-    __version__ ='0.0.3'
-        
-    depends_on = ('raw_records_aqmon') 
-    provides  = ('aqmon_hits')
+    __version__ = '0.0.3'
+
+    depends_on = ('raw_records_aqmon')
+    provides = ('aqmon_hits')
     data_kind = ('aqmon_hits')
-    
+
     dtype = strax.hit_dtype
 
     save_when = strax.SaveWhen.TARGET
 
     def compute(self, raw_records_aqmon):
         rec = strax.raw_to_records(raw_records_aqmon)
         strax.zero_out_of_bounds(rec)
-        strax.baseline(rec, baseline_samples = self.config['baseline_samples_aqmon'], flip = True)
-        aqmon_hits = strax.find_hits(rec, min_amplitude = self.config['hit_min_amplitude_aqmon'])
-    
-        return aqmon_hits 
+        strax.baseline(rec, baseline_samples=self.config['baseline_samples_aqmon'], flip=True)
+        aqmon_hits = strax.find_hits(rec, min_amplitude=self.config['hit_min_amplitude_aqmon'])
 
-    
+        return aqmon_hits
 
 #### Veto hardware ####:
-# V1495 busy veto module: 
-# Generates a 25 ns NIM pulse whenever a veto begins and a 25 ns NIM signal when it ends. 
+# V1495 busy veto module:
+# Generates a 25 ns NIM pulse whenever a veto begins and a 25 ns NIM signal when it ends.
 # A new start signal can occur only after the previous busy instance ended.
 # 1ms (1e6 ns) - minimum busy veto length, or until the board clears its memory
-    
+
 # DDC10 High Energy Veto:
-# 10ms (1e7 ns) - fixed HE veto length in XENON1T DDC10, 
+# 10ms (1e7 ns) - fixed HE veto length in XENON1T DDC10,
 # in XENONnT it will be calibrated based on the length of large S2 SE tails
 # The start/stop signals for the HEV are generated by the V1495 board
-  
+
 @export
 @strax.takes_config(
-    strax.Option('max_veto_gap', default = int(5e8),
-                 help = 'Maximum separation between veto stop and start pulses [ns]'),
-    strax.Option('channel_map', track = False, type = immutabledict, 
-                 help = 'immutabledict mapping subdetector to (min, max)'
+    strax.Option('min_veto_gap', default=1, type=int,
```

Comment:
I've checked the plugin on some recent runs and seems to find the correct deadtimes. I would request one change though. Before you merge, can you please set the min_veto_gap  =  int(1e6) in the default.